tokens/README.md

```diff
@@ -22,3 +22,29 @@ The tokens can be inspected by looking at the source files in the [`properties`
 ### Demo in storybook
 
 We also added an illustrated version of the tokens to the storybook in the [`docs` folder](../docs).
+
+## Getting better editor support for token files
+
+By generating a JSON schema describing the structure of token files and including all token names we make it possible for text editors to provide autocompletion in token JSON files. This is especially useful for token references (e.g. `{font.line-height.style.label.value}`) which would otherwise be typed by hand.
+
+Letting the text editor know which JSON files to apply the schema to requires some configuration.
+
+### VS Code
+
+VS Code has good [documentation](https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings) on getting better editor support for JSON files and working with JSON schemas. TL;DR modify and copy the following into your JSON workspace settings:
+
+```json
+{
+    "json.schemas": [
+        {
+            "fileMatch": [
+                "tokens/properties/**/*.json"
+            ],
+            "url": "./tokens/dist/schema.json"
+        }
+    ]
+}
+```
+
+### JetBrains products
+Go to Settings and navigate to `Languages & Frameworks` > `Schemas and DTDs` >  `JSON Schema Mappings`. There you'll be able to create a mapping for the token files with the schema pointing to `./tokens/dist/schema.json` and the directory set to `tokens/properties/`.
```

Comment:
If possible please move this to a different file other than the README (perhaps somewhere in the storybook docs) As this file is used as our package description on NPM and this information is not relevant to potential consumers of the token package.

Reply:
Thanks, that's a good point! I moved it to "Developing Components" below the "Working with Component-level Design Tokens" section. I don't think that's a perfect fit, but it's the best place I found at the moment. I could imagine splitting the contents of this page up into multiple pages under a more generic "Development" headline at some point in the future.